migrator/README.md

```diff
@@ -0,0 +1,261 @@
+
+```Length (key)``` - the length, of numbers that can be represented in the row
+
+```Old Format``` - represents what digits in the old format need to change, every digit that needs to stay the same is represented as an ‘x’
+E.g. the 3rd index of a 4 digit number needs to be removed/replaced. This number is always 7 → xx7x
+E.g. an addition needs to happen at the 2nd index of a 4 number, so none of the original digits need to be altered → xxxx
+
+```New Format``` - represents what the new number format should be. ‘X’ values represent digits from the original number that will stay the same.
+E.g. digits ‘98’ need to be added at the start of an originally 4 digit number → 98xxxx
+E.g. no digits need to be added because a removal took place. A 5 digit number has now become a 4 digit number → xxxx
+
+```Description``` - text specifying what the change is in words. This will help easily diagnose issues with recipes quickly when the strings in the ‘Old Format’ or ‘New Format’ columns are incorrect.
+
+## Web Application
+
+Please [click here](https://phone-number-migrator.uc.r.appspot.com/) to view the web application version of the migrator tool.
+
+## Disclaimer
+
+The migration library is designed to only migrate E.164 phone numbers using internally maintained migration “recipes”. The library will not attempt to migrate national phone numbers or format non-E.164 phone numbers prior to migrations in order for said numbers to be in E.164 format. Only recipes from the specified country code will be used to perform migrations on a given number(s) input. This means that even if a stale E.164 phone number from country A has a recipe that can be used to migrate it successfully but ```--countryCode=B``` is inputted as the command line argument, the phone number will not be migrated.
+
+For standard migrations using internal recipes, it is possible for E.164 phone numbers to be migrated to a format which the library deems as being invalid. The validity of phone numbers is checked using metadata containing valid number ranges for all countries. As a result, a phone number migration will be seen as invalid if either the metadata of valid number ranges does not reflect the given format correctly or if there is an error with the used recipe internally. In either case, please [file a new issue](https://b.corp.google.com/issues/new?component=192347&template=829703) following the guidance of how to create one [here](https://github.com/google/libphonenumber/blob/master/CONTRIBUTING.md#filing-a-code-issue). in order to resolve the issue. For ```--file``` number inputs, invalid migrations by default will not be written to the created text file containing the updated phone numbers and instead, the original number which correlates to the migration will be used. If you would like to include invalid migration results in the newly created text file, specify this by using the ```--exportInvalidMigrations``` command line flag.
```

Comment:
I am not sure the period should be after the link of how to create one? before 'in order to resolve the issue'